### PR TITLE
[DEV-6622] COVID Page Query Param "section" Restored

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**

Restores the query param "section" overwritten by the "publicLaw" param

**Technical details:**

See tests and commits

**JIRA Ticket:**
[DEV-6622](https://federal-spending-transparency.atlassian.net/browse/DEV-6622)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
`N/A` Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
`N/A` Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
`N/A` Verified mobile/tablet/desktop/monitor responsiveness
`N/A` Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [x] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
`N/A` [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
`N/A` [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
`N/A` Design review complete `if applicable`
`N/A` [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/sharedComponents/GlossaryLink.jsx [hunk 1]
```diff
@@ -11,8 +11,7 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { getNewUrlForGlossary } from 'helpers/glossaryHelper';
 
 const propTypes = {
-    term: PropTypes.string.isRequired,
-    currentUrl: PropTypes.string.isRequired
```
Comment (ebdabbs) [inline src/js/components/sharedComponents/GlossaryLink.jsx:15]: Thanks!

File: src/js/containers/covid19/Covid19Container.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import React, { useEffect, useRef } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 import GlobalConstants from 'GlobalConstants';
-import { useQueryParams } from 'helpers/queryParams';
+import { combineQueryParams, getQueryParamString, useQueryParams } from 'helpers/queryParams';
```
Comment (ebdabbs) [inline src/js/containers/covid19/Covid19Container.jsx:10]:
```suggestion
import { getQueryParamString, useQueryParams } from 'helpers/queryParams';
```
(`no-unused-vars` lint error)

File: tests/containers/covid19/Covid19Container-test.jsx [hunk 3]
```diff
@@ -26,7 +26,15 @@ jest.mock('containers/covid19/WithDefCodes', () => ({
 
 // Mock the custom hook useQueryParams
 jest.mock('helpers/queryParams', () => ({
-    useQueryParams: jest.fn()
+    useQueryParams: jest.fn(),
+    getQueryParamString: jest.fn().mockImplementation((obj) => {
```
Comment (ebdabbs) [inline tests/containers/covid19/Covid19Container-test.jsx:30]: Could we use the real function or just mock the return value in the tests below? It seems counter-intuitive to recreate this functionality in our test file. In other words, when we "expect" a certain search string below, it is testing this implementation instead what actually gets run by the app.

**Review by ebdabbs — CHANGES_REQUESTED** [review]

File: tests/containers/covid19/Covid19Container-test.jsx (hunk 3 above)
Comment (MaxwellKendall) [reply tests/containers/covid19/Covid19Container-test.jsx:30]: Its a built in function on the browser and doesnt exist in the test env.

**Review by ebdabbs — APPROVED** [review]

**Pull request merged**